Cargo.toml

```diff
@@ -18,6 +18,9 @@ clap = "2.33.3"
 regex = "1"
 procfs = "0.8.1"
 actix-web = "3"
+riemann_client = "0.7.0"
```

Comment:
I never tried it but you also have https://github.com/sunng87/rustmann which supports stuff like mTLS.

Reply:
In your experience is this a must have or a nice to have for a riemann client ?

Reply:
Interesting because based on tokio as well, but maybe a bit early to use this client. As stated in the presentation.
> A riemann client using tokio. This project is still in its early stage and API changes aggressively.

So we may run to adapt the code to the api changes. I think we can keep an eye on it and move as soon as the api is more stable and if it makes sens.

Reply:
> In your experience is this a must have or a nice to have for a riemann client ?

it's not mandatory but I usually run all of my services using mTLS if I can.
But as @uggla said, the current lib may be enough for now.

Reply:
Thanks a lot for your feedbacks. Let's keep this one for now and keep an eye on rustmann :)

Reply:
Another option could be to add mTLS to the current riemann client using rustls. Seems not "too hard" although with security it is never as easy as it looks...